lib/curl_trc.h

```diff
@@ -145,6 +150,9 @@ void Curl_trc_cf_infof(struct Curl_easy *data, struct Curl_cfilter *cf,
 #error "missing VARIADIC macro define, fix and rebuild!"
 #endif
 
+#define infog(...)  Curl_nop_stmt
```

Comment:
Can't we just make `infof()` require double parentheses? I would prefer sticking to the same name we have used for so long (introduced before year 2000), it is now stuck in my muscle memory.

Reply:
It was just temporary to allow upgrading them gradually (and allow testing with the first batch of changes).
Now that I finished all of them, it can be renamed back to its usual name. I'll do that soon.